app/src/main/java/mozilla/lockbox/store/LockedStore.kt

```diff
@@ -14,11 +15,19 @@ import mozilla.lockbox.flux.Dispatcher
 open class LockedStore(
     val dispatcher: Dispatcher = Dispatcher.shared
 ) {
+    lateinit var keyguardManager: KeyguardManager
+
     companion object {
         val shared = LockedStore()
     }
 
     open val onAuthentication: Observable<FingerprintAuthAction> =
         dispatcher.register
             .filterByType(FingerprintAuthAction::class.java)
+
+    fun apply(manager: KeyguardManager) {
+        keyguardManager = manager
+    }
+
+    val isKeyguardSecure get() = keyguardManager.isKeyguardSecure
```

Comment:
Rather than having this in the `LockedStore`, I'd put this in the `FingerprintStore` where we are checking other device security settings (corresponds to @linuxwolf 's desire to have all device security information in one place)

It means passing two things to the `FingerprintStore.apply()` method which is 🤮 but will get fixed up by #177